frontend/src/Student/components/DisplayNameInput.js

```diff
@@ -5,13 +5,57 @@ import FontAwesomeIcon from "@fortawesome/react-fontawesome";
 import FaQuestionCircle from "@fortawesome/fontawesome-free-solid/faQuestionCircle";
 
 export default function DisplayNameInput({
-  values,
```

Comment:
I think you need to leave this in, I was getting an error without it. Otherwise everything seems to work

Reply:
Fixed